File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/android/SharedPreferencesInjector.java
package com.savvasdalkitsis.gameframe.injector.android;

import android.content.SharedPreferences;
import android.preference.PreferenceManager;

import static com.savvasdalkitsis.gameframe.injector.ApplicationInjector.application;

public class SharedPreferencesInjector {

    public static SharedPreferences sharedPreferences() {
        return PreferenceManager.getDefaultSharedPreferences(application());
    }

}

File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/android/SharedPreferencesInjector.kt
package com.savvasdalkitsis.gameframe.injector.android

import android.content.SharedPreferences
import android.preference.PreferenceManager

import com.savvasdalkitsis.gameframe.injector.ApplicationInjector.application

object SharedPreferencesInjector {

    fun sharedPreferences(): SharedPreferences =
            PreferenceManager.getDefaultSharedPreferences(application())

}
